src/utils/sortByRelevance.tsx

```diff
@@ -0,0 +1,21 @@
+import { sortBy } from "lodash";
+
+/**
+ * Sorts an array of strings based on their relevance to a target string.
+ *
+ * @param items An array of strings to sort
+ * @param target The target string to compare against
+ * @returns A new array of sorted strings
+ */
+export function sortByRelevance(items: string[], target: string): string[] {
```

Comment:
as you've created `utils`, I'd suggest to have a generic component which can be grown later. So instead of  `sortByRelevance.tsx` as standalone component, which when you import the method, it looks like

```javascript
import { sortByRelevance } from "../../utils/sortByRelevance";
import { newUtilMethod } from "../../utils/newUtilMethod";
import { anotherUtilMethod } from "../../utils/anotherUtilMethod";
.
.
.
import { oneMoreUtilMethod } from "../../utils/oneMoreUtilMethod";
```

please create something like `helpers.tsx` or `commonUtils.tsx` or whatever, normally, it'd be called `helpers`. And when you import such file, it looks like this

```javascript
import { sortByRelevance, newUtilMethod, anotherUtilMethod, ...   } from "../../utils/helper";
```

Reply:
That is another good way to organise the utils. Basically now I just fowllow current the file structure , I found there is already a utils folder for different utils functions.